charts/postgres-operator/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+name: postgres-operator
+version: 0.1.0
+appVersion: 1.0.0-37-g2422d72
```

Comment:
You probably need to update PR, since the current release of postgres-operator is 1.1.0

Reply:
fixed